### PR TITLE
Set thumbnail size from CSS

### PR DESCRIPTION
**Pull request by protyposis**
The thumbnail size was set programmatically in pixels and was directly taken from the sprites file, so it could only be changed by adjusting the sprites file. 

Now the thumbnail size can be a relative or absolute size. It's not programmatically set any more so it can be set in CSS too.

### CODE REVIEW

File: src/ts/imageloader.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        image: new DOM('img', {}),
+        loadedCallback: loadedCallback,
+        loaded: false,
+        width: 0,
+        height: 0,
+      };
+      this.state[url] = state;
+
+      // We need to add the image element to the DOM to get the size of the loaded image
+      new DOM('body').append(state.image);
```
Comment (HalfbyteHeroes) [inline src/ts/imageloader.ts]: I think we should set the `display` of the img-tag to `none` before appending it to the body to make sure that it is never visible to the user, even when it is directly removed in `onload`.

Comment (protyposis) [inline src/ts/imageloader.ts]: That does not work. The element is never rendered and width/height stay zero. The whole purpose of this loader is to get the image size. I checked it in every browser and the image is never visible.

Comment (HalfbyteHeroes) [inline src/ts/imageloader.ts]: Have you tried something like [this](https://stackoverflow.com/questions/11442712/get-width-height-of-remote-image-from-url)?
```javascript
function getMeta(url, callback) {
    var img = new Image();
    img.src = url;
    img.onload = function() { callback(this.width, this.height); }
}
getMeta(
  "http://snook.ca/files/mootools_83_snookca.png",
  function(width, height) { alert(width + 'px ' + height + 'px') }
);
```

Comment (protyposis) [inline src/ts/imageloader.ts]: Thanks, works nicely :)

**Pull request merged**